calendar/calendar.yaml

```diff
@@ -38,6 +41,29 @@
       Meeting Host Link: https://zoom.us/s/83583392870
   organizer: autobot@kubeflow.org
 
+- id: kf002
```

Comment:
Can you increment this number instead (kf033 will be the next id)? I am not sure what will happen if you reuse an old event id.

Reply:
@james-jwu I actually verified that an event with ID `kf002` does not exist on that calendar.

Reply:
It existed is a previous commit: https://github.com/kubeflow/community/pull/283

Just out of caution can we use an unused event id?

Reply:
Updated to `kf033` in https://github.com/kubeflow/community/pull/575/commits/49a21eb09838f5275fc44f1f962685b3f7498d27